Data/ByteString.hs

```diff
@@ -274,9 +275,28 @@ import GHC.Word hiding (Word8)
 -- -----------------------------------------------------------------------------
 -- Introducing and eliminating 'ByteString's
 
+allBytes :: ByteString
+allBytes = unsafePackLenLiteral 0x100
+  "\x00\x01\x02\x03\x04\x05\x06\x07\x08\x09\x0a\x0b\x0c\x0d\x0e\x0f\
```

Comment:
Please format it into one long line, otherwise any attempt to add `{-# LANGUAGE CPP #-}` will cause havoc.

Reply:
One long line is harder to visually inspect, but I guess it doesn't really matter. A typo causes around two dozen test failures anyway.

Coming right up.